### DISCUSSION TITLE
Want to +1 hour to the output time

### DISCUSSION THREAD

**Discussion by SENPAY98K** [moment/moment#6429] [Q&A] [answered]
Hi, how do i add +1 to hours ?? iam on UTC+1 but seems not to get my local time

Iam using this: `var time = moment().format('yyyy-MM-DD, HH:mm');`
The output i.e: `2021-06-07, 23:02`

but i want to add +1 to hours to match my local time
ie: `2021-06-07, 00:02`

**Comment by santhoshjanan** [comment]
You can use 

`moment().add(1, 'hour').format('yyyy-MM-DD, HH:mm')`

Or check: [this stalkblitz](https://stackblitz.com/edit/js-yx1rvx?devtoolsheight=33&file=index.js)

**Comment by santhoshjanan** [comment]
Another solution is to use [Moment-Timezone](https://momentjs.com/timezone/docs/)

```
var moment = require('moment-timezone');
moment().tz("America/Los_Angeles").format();
```

**Comment by itsdeekay** [comment]
@SENPAY98K Is issue resolved?

**Comment by mattjohnsonpint** [comment, marked as answer]
Thanks for the question! A literal adjustment is `.add(1, 'hour')`, including any date rollover. For actual timezone conversion, I would not add a fixed hour because DST changes offsets; I'd use [Moment Timezone](https://momentjs.com/timezone/docs/) with a named zone instead.
